src/textures/TextTextureRenderer.mjs

```diff
@@ -293,7 +294,17 @@ export default class TextTextureRenderer {
         this._context.fillStyle = StageUtils.getRgbaString(this._settings.textColor);
         for (let i = 0, n = drawLines.length; i < n; i++) {
             let drawLine = drawLines[i];
-            this._context.fillText(drawLine.text, drawLine.x, drawLine.y);
+
+            if (!letterSpacing) {
```

Comment:
I would prefer `letterSpacing == 0` instead for better readability.

Reply:
updated